README.md

```diff
@@ -478,6 +479,8 @@ enum L10n {
 ```
 </details>
 
+</details>
+
```

Comment:
Duplicate closing `</ details>` tag here?

Reply:
sorry...it slipped in the commit